tools/make.py

```diff
@@ -1302,13 +1302,13 @@ def main(argv):
                     addonTomlPath = os.path.join(work_drive, prefix, module, "addon.toml")
                     if os.path.isfile(addonTomlPath):
                         with open(addonTomlPath, "r") as f:
-                            skipPreprocessing = "preprocess = false" in f.read() #python 3.11 has real toml but this is fine for now
+                            skipPreprocessing = "[preprocess]\nenabled = false" in f.read() #python 3.11 has real toml but this is fine for now
```

Comment:
```suggestion
                            skipPreprocessing = "[preprocess]\nenabled = false" in f.read() or "preprocess = false" in f.read()
                            #python 3.11 has real toml but this is fine for now
```

Reply:
What is the purpose of including this if it won't be used any more?

Reply:
I'd preferer to keep backward compatibility
I use these tools for other projects

Reply:
This is being deprecated in HEMTT as it was an outlier in the way it was formatted, it'll eventually be removed, you should update your other projects